### PR TITLE
Use Docker for local testing

### PR DESCRIPTION
**Pull request by eagletmt**
I added a Dockerfile to run local test easily.
It also provides example configurations for each component (master, web, worker, client).

### CODE REVIEW

File: local_test/run_worker.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -ex
+
+mkdir -m700 -p /root/.ssh
+cp local_test/id_rsa.rrrspec /root/.ssh/id_rsa
```
Comment (draftcode) [inline local_test/run_worker.sh]: I got a complaint about id_rsa's permission. Don't we need to chmod 600 id_rsa?

Comment (eagletmt) [inline local_test/run_worker.sh]: id_rsa.rrrspec in the repository is 0600 and `cp` keeps the file mode when it creates a new file.

Comment (eagletmt) [inline local_test/run_worker.sh]: Added `chmod 600` and `-p` flag.
https://github.com/cookpad/rrrspec/commit/8c76d06de45e6fb7093f84d6898b9d9fe5ab06c0

Comment (draftcode) [inline local_test/run_worker.sh]: Git preserves only an executable bit. Even if you commit a file in 600, it will be checked out as 644.

**Comment by draftcode** [comment]
LGTM

**Comment by eagletmt** [comment]
I found RRRSpec doesn't work even with RSpec 2.99.x :bomb:

**Comment by draftcode** [comment]
:facepunch:

File: local_test/run_worker.sh (hunk 1 above)
Comment (eagletmt) [inline local_test/run_worker.sh]: naruhodo-

**Pull request merged**